src/app/back/manager/BookManager.php

```diff
@@ -33,4 +38,32 @@ public function all(): array
         return $stmt->fetchAll(PDO::FETCH_CLASS, Book::class);
     }
 
+    /**
+     * @param int $id
+     * @return Book|null
+     */
+    public function one(int $id): ?Book
+    {
+        $stmt = $this->db->prepare('SELECT * FROM book WHERE id = :id');
+        var_dump($id);
+        $stmt->execute([':id' => $id]);
+        $stmt->setFetchMode(PDO::FETCH_CLASS, Book::class);
+        return $stmt->fetch();
+    }
+
+    /**
+     * @param Book $book
+     * @return Book|null
+     */
+    public function insert(Book $book): ?Book
+    {
+        $stmt = $this->db->prepare('INSERT INTO book VALUES (:id, :name, :publisher, :price)');
+        $result = $stmt->execute([
+            ':id' => $book->getId(),
+            ':name' => $book->getName(),
+            ':publisher' => $book->getPublisher(),
+            ':price' => $book->getPrice()]);
```

Comment:
C'est peut être plus lisible d'écrire : 
```suggestion
        $result = $stmt->execute([
            ':id' => $book->getId(),
            ':name' => $book->getName(),
            ':publisher' => $book->getPublisher(),
            ':price' => $book->getPrice()
        ]);
```

Reply:
Oui, en effet, c'était comme ça avant. J'ai du faire une fausse manip.

Reply:
Tu l'as commit ? Je ne le vois pas !